### PR TITLE
ci: Run UI Tests of Open Source Apps

### PR DESCRIPTION
**Pull request by philipphofmann**
Pull the repository of [VLC](https://github.com/videolan/vlc-ios) and [Home Assistant](https://github.com/home-assistant/iOS) and run their UI tests to ensure the APM of the Cocoa SDK doesn't introduce any major issues.

Fixes GH-1513

#skip-changelog

### CODE REVIEW

**Review by brustolin — APPROVED** [review]
This is awesome! Will help a lot!!

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -0,0 +1,164 @@
+name: Integration Tests
+on:
+  push:
+    branches:
+      - master
+      - ci/**
```
[the remark this answers is not in the document]
Comment (bruno-garcia) [reply .github/workflows/integration-tests.yml:6]: seems it was forgotten

**Review by bruno-garcia — COMMENTED** [review]
very cool

**Comment by philipphofmann** [comment]
> seems it was  forgotten

🤓 Already fixed https://github.com/getsentry/sentry-cocoa/pull/1556/files

**Pull request merged**